homeassistant/components/derivative/sensor.py

```diff
@@ -116,18 +116,14 @@ async def async_setup_entry(
     else:
         device_info = None
 
-    unit_prefix = config_entry.options[CONF_UNIT_PREFIX]
-    if unit_prefix == "none":
-        unit_prefix = None
```

Comment:
This can still be `"none"` from a loaded ConfigEntry that was created before this change. It would raise a `KeyError` then here https://github.com/home-assistant/core/blob/d13d4feca30e0fcfd61be645f7a2879b10158672/homeassistant/components/derivative/sensor.py#L194

See also #102257

Reply:
Fixed in https://github.com/home-assistant/core/pull/102294